internal/website/content/pages/pubsub.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Pubsub"
+title: "Pub/Sub"
 ---
 
-Pubsub refers to implementations of the [publish-subscribe
+Pub/Sub refers to implementations of the [publish-subscribe
```

Comment:
Should it be publish-subscribe or publisher subscriber (change in another file in this PR)? The wikipedia link does the former

Reply:
The rule I had in my head was "Publisher/Subscriber" is a standalone noun, "publish-subscribe model" is a different noun.